data_collection/gazette/spiders/sp_araraquara.py

```diff
@@ -0,0 +1,57 @@
+import datetime
+from urllib.parse import urlencode
+
+import dateparser
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class SpAraraquaraSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3503208"
+    name = "sp_araraquara"
+    allowed_domains = ["diariooficialcmararaquara.sp.gov.br"]
+    start_date = datetime.date(2021, 3, 4)  # First gazette available
+    end_date = datetime.datetime.today()
+    start_urls = ["https://www.diariooficialcmararaquara.sp.gov.br"]
+    search_url = "https://www.diariooficialcmararaquara.sp.gov.br/diario/busca?"
+    base_url = "https://www.diariooficialcmararaquara.sp.gov.br"
+
+    def parse(self, response):
+
+        initial = self.start_date.strftime("%d/%m/%Y")
+        end = self.end_date.strftime("%d/%m/%Y")
+
+        params = {
+            "data": initial,
+            "dataFinal": end,
+            "descricao": "",
+            "subcategoria": "",
+        }
+        url_params = urlencode(params)
+        yield response.follow(f"{self.search_url}{url_params}", self.parse_gazette)
+
+    def parse_gazette(self, response):
+
+        gazettes = response.css(".event-card.animated.flipInX")
```

Comment:
**suggestion** Everything that has class `event-card` is a gazette, so you can replace it as ```gazettes = response.css(".event-card")```